### PR TITLE
Added initial support for collections

### PR DESCRIPTION
**Pull request by rudymalhi**

### CODE REVIEW

File: shoebox/public/site/js/jquery-ui-1.10.3.custom/index.html [hunk 1]
```diff
@@ -0,0 +1,301 @@
+<!doctype html>
```
Comment (gmethvin) [inline shoebox/public/site/js/jquery-ui-1.10.3.custom/index.html]: I'd rather not commit code/pages that we don't need. I think we just need the minified css and js files for jQuery UI.

Comment (rudymalhi) [inline shoebox/public/site/js/jquery-ui-1.10.3.custom/index.html]: Fixed

Comment (gmethvin) [inline shoebox/public/site/js/jquery-ui-1.10.3.custom/index.html]: Do we still need this index.html file?

**Comment by gmethvin** [comment]
Feel free to merge after removing the extra file.

**Comment by rudymalhi** [comment]
@gmethvin, I added some new changes, please check and merge

**Pull request merged**